x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/properties/build_fields.ts

```diff
@@ -108,6 +108,9 @@ export const getFields = (metadata: InfraMetadata, group: 'cloud' | 'host' | 'ag
           name: 'agent.policy',
           value: metadata.info?.agent?.policy,
         },
-      ];
+      ]);
   }
 };
+
+const prune = (fields: Array<{ name: string; value: string | string[] | undefined }>) =>
+  fields.filter((f) => !!f.value);
```

Comment:
I usually like doing `filter(Boolean)` for added clarity, since `!!` can sometimes be mistaken for a typo, but this is fine